frontend/src/components/Menu/Sidebar.vue

```diff
@@ -49,7 +49,7 @@
           </b-nav-item>
           <b-nav-item
             class="mb-3 text-light"
-            v-if="$store.state.roles.length > 0"
+            v-if="$store.state.roles && $store.state.roles.length > 0"
```

Comment:
@Mogge roles are not commited to the store so this should always be false or do I miss something?
```suggestion
            v-if="$store.state.moderator?.roles?.length > 0"
```